stdlib/public/SwiftRemoteMirror/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 if(SWIFT_BUILD_DYNAMIC_STDLIB)
   add_swift_target_library(swiftRemoteMirror
                            SHARED DONT_EMBED_BITCODE NOSWIFTRT
+                           ../runtime/LLVMSupport.cpp
```

Comment:
Not sure if this is okay to do...

Reply:
mmm... can the remote mirror library not link against LLVMSupport?

Reply:
@dcci @jckarter Would it be okay to link swiftRemoteMirror against LLVMSupport?

Reply:
Should be fine to just link LLVMSupport. That'd also let us get rid of the iostream uses in favor of raw_ostream.

Reply:
I thought we were worried about it getting loaded into a process that had its own LLVMSupport.

Reply:
I think as long as we make sure it gets linked in with hidden visibility, it should be OK.